TDP/tdp-53.md

```diff
@@ -0,0 +1,110 @@
+It is proposed that $30,741.68 is transferred in full prior to the start of the funding round (January 1st, 2023).
+
+This will allow the subDAO funding to be used immediately for expenses requiring advance payment, and help streamline the accounting process over the course of the funding period.
+The mStableDAO is expected to hold the subDAO accountable for using funds as outlined and for meeting deliverables over the efunding period as defined in this proposal. This process will include a monthly review of the Ecosystem subDAO's activities and performance.
+
+Below is an itemized list of expected costs. Although not anticipated, the buffer amount will provide some flexibility if there is a variance in expenditures beyond the baseline budget outlined herein.
+
+**Itemized Budget**
+
+- Salaries (Total)	- $25,066.68
+- mStable Merchandise - $2,000
+- Discord Server Costs - $175
+- Signer Gas Costs - $500
+- Buffer Amount - $3,000
+
+**Total mUSD Requested:	$30,741.68**
+
+The aforementioned budget represents a $2,000 mUSD budget reduction (coming from merchandise) when compared to the previous funding round.
+
+All unallocated budget after 4 months will be returned to the TreasuryDAO or at the discretion of the TreasuryDAO, may be subtracted from any subsequent approved funding request. Note that any calculation of unallocated budget should take into account any outstanding accounts payable to ensure that the subDAO can meet future payment obligations.
+
+To ensure that the subDAO is not incentivized to overspend and use the buffer amount, the TreasuryDAO should hold the Ecosystem subDAO accountable to the original budget and consider use of the buffer as a negative indicator of performance.
+
+### MTA
+
+The full amount requested for the period is 83125 MTA, but with 65774 left over from the previous funding round. Therefore, the additional amount requested is only 17.351.
+
+The amount of MTA requested will be utilized primarily to cover Dework Bounties (particularly for mStable v1), moderator pay, community events, and any other community-based initiatives after the launch of Meta Vaults
+
+Any unallocated MTA at the end of the period will be returned to the TreasuryDAO or subtracted from the following funding period's requested total.
+
+Below is an itemized list of expected costs in MTA for the funding round. The 17,351 MTA will bring the Ecosystem subDAO's MTA total to 83,125. Similar to the last round, the Ecosystem subDAO will act judiciously with funds allocated for discretionary activities and operate in a lean manner. Please note that moderator pay will be denominated in MTA.
+
+**Itemized Budget**
+
+- Dework Bounties - 30,000
+- Community Events - 18,750
+- Moderator Pay: 25,000
+- Raffles - 6,250
+- Rewards for Completing Surveys - 1,875
+- Tips - 1250
+
+Summary:  83,125 MTA amount with **17,351** requested for this funding round
+
+**Total MTA Requested: 17,351**
+
+## Deliverables

```

Comment:
This is within specifications or? should been h3?

Reply:
Yes true